rma_sale/wizards/rma_make_sale_order.py

```diff
@@ -166,11 +178,17 @@ class RmaLineMakeSaleOrderItem(models.TransientModel):
 
     def _compute_line_id(self):
         rma_line_obj = self.env["rma.order.line"]
+        rma_obj = self.env["rma.order"]
+        active_ids = self.env.context.get("active_ids") or []
+        active_model = self.env.context.get("active_model")
         for rec in self:
-            if not self.env.context["active_ids"]:
+            if not active_ids:
                 return
-            rma_line_ids = self.env.context["active_ids"] or []
-            lines = rma_line_obj.browse(rma_line_ids)
+            if active_model == "rma.order":
+                rma = rma_obj.browse(active_ids)
+                lines = rma.rma_line_ids
+            else:
+                lines = rma_line_obj.browse(active_ids)
```

Comment:
I don't understand this whole method. Is this really used ? I think it could be removed fully...
It is a compute method from line_id field to link the item line with the rma_line, but on the loop, it will always take the same rma_line right ? (the first one in active_ids/rma order). So this method assign all item line to the same rma_line.

Then in default_get it is well set...I guess since it is set on default_get, this _compute_line_id is kind of useless, that is why it is working... I'd remove it totally...